NEWS.rst

```diff
@@ -1,6 +1,13 @@
 History
 =======
 
+0.9 (2019-10-02)
+----------------
+* Add Antenna.reference_antenna utility function (#51)
```

Comment:
We ended up calling it `array_referece_antenna`.

Reply:
Good point!